src/frontend/components/DashboardVideoLiveControlPane/customs/SharedLiveMediaModalWrapper/index.spec.tsx

```diff
@@ -0,0 +1,133 @@
+  useDeleteSharedLiveMediaUploadModal: jest.fn(),
+}));
+
+const mockUseDeleteSharedLiveMediaUploadModal =
+  useDeleteSharedLiveMediaUploadModal as jest.MockedFunction<
+    typeof useDeleteSharedLiveMediaUploadModal
+  >;
+
+describe('<SharedLiveMediaModalWrapper />', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders nothing if there is no shared live media to delete', () => {
+    mockUseDeleteSharedLiveMediaUploadModal.mockReturnValue([null, jest.fn()]);
+    render(<SharedLiveMediaModalWrapper />);
+    expect(screen.queryByText('Delete shared media')).toBe(null);
+    expect(screen.queryByRole('button', { name: 'Confirm' })).toBe(null);
+    expect(screen.queryByRole('button', { name: 'Cancel' })).toBe(null);
+  });
+
+  it('renders the modal when there is a shared live media to delete, and clicks on cancel', async () => {
+    const mockedSharedLiveMedia = sharedLiveMediaMockFactory();
+    const mockSetDeleteSharedLiveMediaUploadModal = jest.fn();
+    mockUseDeleteSharedLiveMediaUploadModal.mockReturnValue([
+      mockedSharedLiveMedia,
+      mockSetDeleteSharedLiveMediaUploadModal,
+    ]);
+    render(<SharedLiveMediaModalWrapper />);
+    screen.getByText('Delete shared media');
+    screen.getByText(
+      `Are you sure you want to delete file ${mockedSharedLiveMedia.title} ?`,
+    );
+    screen.getByRole('button', { name: 'Confirm' });
+    const cancelButton = screen.getByRole('button', { name: 'Cancel' });
+    act(() => userEvent.click(cancelButton));
+    expect(mockSetDeleteSharedLiveMediaUploadModal).toHaveBeenCalledTimes(1);
+    expect(mockSetDeleteSharedLiveMediaUploadModal).toHaveBeenCalledWith(null);
+  });
+
+  it('renders the modal when there is a shared live media to delete, and clicks on confirm', async () => {
+    const mockedSharedLiveMedia = sharedLiveMediaMockFactory();
+    mockUseDeleteSharedLiveMediaUploadModal.mockReturnValue([
+      mockedSharedLiveMedia,
+      jest.fn(),
+    ]);
+    fetchMock.delete(`/api/sharedlivemedias/${mockedSharedLiveMedia.id}/`, 204);
+    const mockRemoveResource = jest.fn();
+    useSharedLiveMedia.getState().removeResource = mockRemoveResource;
+
+    render(<SharedLiveMediaModalWrapper />);
+    screen.getByText('Delete shared media');
+    screen.getByText(
+      `Are you sure you want to delete file ${mockedSharedLiveMedia.title} ?`,
+    );
+    screen.getByRole('button', { name: 'Cancel' });
+    const confirmButton = screen.getByRole('button', { name: 'Confirm' });
+    act(() => userEvent.click(confirmButton));
+    await waitFor(() =>
+      expect(fetchMock.lastCall()![0]).toEqual(
+        `/api/sharedlivemedias/${mockedSharedLiveMedia.id}/`,
+      ),
+    );
+    expect(fetchMock.lastCall()![1]).toEqual({
+      headers: {
+        Authorization: 'Bearer json web token',
+        'Content-Type': 'application/json',
+      },
+      method: 'DELETE',
+    });
+    screen.getByText('Shared media updated.');
```

Comment:
`updated` does not seems appropriated. `Deleted` is better IMO.

Reply:
done